docs/source/pages/installation.rst

```diff
@@ -0,0 +1,13 @@
+============
+Installation
+============
+
+Install from pypi using::
+
+    pip install sanic-jwt
+
+Install from GitHub::
```

Comment:
Would `pip install git+https ...` be an alternative?

Reply:
I think I want to take that out. Pypi should be the recommended method so we can control releases. people that would install that way I presume would know what they are getting into.

Reply:
I think that's a fair point :smile: